commitlint.config.js

```diff
@@ -7,7 +7,9 @@ const Configuration = {
             'always',
             ['build', 'ci', 'docs', 'feat', 'fix', 'style', 'refactor', 'test', 'revert', 'chore']
         ],
-        'scope-enum': [2, 'always', ['Core', 'Platform', 'doc-app', 'release', 'e2e', 'deps', 'deps-dev']]
+        'scope-enum': [2, 'always', ['Core', 'Platform', 'doc-app', 'release', 'e2e', 'deps', 'deps-dev']],
```

Comment:
Will it work for `(Core|Platform)` cases?

Reply:
 Why do we have `Core`, `Platform` starting from uppercase and others not? 
I supposed that the scope should always start from uppercase.

Reply:
> Will it work for `(Core|Platform)` cases?

yes

Reply:
>  Why do we have `Core`, `Platform` starting from uppercase and others not?
> I supposed that the scope should always start from uppercase.

part of these scopes generated through bots and I can not influence this
that is why we have part of these scopes lowercase